src/codegen/spirv/ir_builder.cc

```diff
@@ -222,27 +222,27 @@ Value IRBuilder::GetPushConstant(
   return this->MakeValue(spv::OpLoad, v_type, ptr);
 }
 
-Value IRBuilder::DeclareKernelFunction(const std::string& name,
-                                       bool uses_workgroup_id,
-                                       bool uses_local_id) {
-  if (uses_workgroup_id) {
-    GetWorkgroupID(0);  // Ensure workgroup_id_ is valid
-  }
-  if (uses_local_id) {
-    GetLocalID(0);  // Ensure local_id_ is valid
-  }
+Value IRBuilder::NewFunction() {
+  return NewValue(t_void_func_, kFunction);
+}
+
+void IRBuilder::CommitKernelFunction(const Value& func_id, const std::string& name) {
+  //if (uses_workgroup_id) {
```

Comment:
please remove comments

Reply:
Done, noticed those just after committing. They are removed from the latest commit